### DISCUSSION TITLE
Backup strategy

### DISCUSSION THREAD

**Discussion by schoentoon** [ArchiveBox/ArchiveBox#841] [Q&A]
What would be the best strategy to creating a backup of your archivebox instance? It containing a sqlite database makes me believe just straight creating a complete copy, including the sqlite database while archivebox is still running might not be the best strategy. Would it be wiser to simply stop the process, create a full backup including the sqlite database. Or would dumping the sqlite database into it's own file beforehand (while archivebox is still running) be wiser?

**Comment by pirate** [comment]
Just stop the whole process and zip up the entire data folder. Everything needed is inside. No special treatment / dumping of the SQLite db is necessary as long as it's paused while backing up.

**Comment by sigdevel** [comment]
Have there been any new methods of “proper copying” recently?

**Reply by pirate** [reply]
Nope, just save the entire `./data` folder and you're good to go.

There is no state kept outside that folder.

**Reply by sigdevel** [reply]
> Nope, just save the entire `./data` folder and you're good to go.
> 
> 
> 
> There is no state kept outside that folder.

That's acceptable too. thank you (:
